tests/test_cli.py

```diff
@@ -21,13 +21,13 @@ def test_doi_to_be_added_false():
 
 
 def test_doi_to_be_retried_true():
-    doi_items = [{"doi": "111.1/111", "status": "Failed, will retry"}]
+    doi_items = [{"doi": "111.1/111", "status": str(Status.FAILED.value)}]
```

Comment:
What happens if you leave the str()?

Reply:
I'm storing the numeric value as a `str` in DynamoDB because storing it as an `int` causes DynamoDB to export as a `Decimal` type which causes complications in other parts. To avoid an import and extra processing related to that, I'm just wrapping the numeric value as a `str` in the tests

Reply:
Ok, we can address this later if necessary